omnibus/config/software/fixie-gem.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 name "chef_fixie"
-default_version "0.4.0"
+default_version "master"
```

Comment:
Yeah, this is probably for the best. It's not like the update rate of fixie is high.